### DISCUSSION TITLE
Navigation

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#39754] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

What is Navigation?

**Comment by DSDmark** [comment]
Hi @iLakshya,
I guess, in technical term, navigation is used to navigate somewhere. As I have knowledge, navigation is also using to operate something. I know my English is not good 🥲

**Comment by look** [comment, marked as answer]
@iLakshya in the context of this product feature category, it refers to code navigation like in an editor or IDE. For example: jump to function definition or find all references for a symbol.
